check/check.go

```diff
@@ -242,7 +243,7 @@ func isShellCommand(s string) bool {
 
 	out, err := cmd.Output()
 	if err != nil {
-		fmt.Fprintf(os.Stderr, "%s\n", err)
+		fmt.Fprintf(os.Stderr, "failed to check if command: %q is valid %v\n", s, err)
 		os.Exit(1)
```

Comment:
NIT: To facilitate testing I'd return a wrapped error rather than call os.Exit here.